website/src/pages/index.js

```diff
@@ -0,0 +1,297 @@
+// /**
+//  * Copyright (c) 2017-present, Facebook, Inc.
+//  *
+//  * This source code is licensed under the MIT license found in the
+//  * LICENSE file in the root directory of this source tree.
+//  */
+
+import React, { useState, useEffect } from 'react';
+
+import CodeBlock from '@theme/CodeBlock';
+import Heading from '@theme/Heading';
+import Jump from '@site/src/components/Jump';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import SVG from 'react-inlinesvg';
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import classnames from 'classnames';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+
+import styles from './index.module.css';
+import './index.css';
+
+const AnchoredH2 = Heading('h2');
+
+const features = [
+  {
+    title: 'Blazing Fast',
+    icon: 'zap',
+    description: (
+      <>
+        Pinot is designed to answer OLAP queries with low latency on immutable data
+      </>
+    ),
+  },
+  {
+    title: 'Pluggable indexing',
+    icon: 'unlock',
+    description: (
+      <>
+        Pluggable indexing technologies - Sorted Index, Bitmap Index, Inverted Index
+      </>
+    ),
+  },
+  {
+    title: 'Near Real time ingestion',
+    icon: 'rss',
+    description: (
+      <>
+        Near Realtime ingestion with <Link to="https://kafka.apache.org/">Apache Kafka</Link> supports StringSerializer or <Link to="https://avro.apache.org/">Avro</Link> formats
+      </>
+    ),
+  },
+  {
+    title: 'Horizontally scalable',
+    icon: 'code',
+    description: (
+      <>
+        Horizontally scalable and fault tolerant
+      </>
+    ),
+  },
+  {
+    title: 'Joins using PrestoDB',
+    icon: 'shuffle',
+    description: (
+      <>
+        Joins are currently not supported, but this problem can be overcome by using <Link to="https://prestodb.io/">PrestoDB</Link> for querying
+      </>
+    ),
+  },
+  {
+    title: 'SQL-like Query Interface (PQL)',
+    icon: 'search',
+    description: (
+      <>
+        SQL like language that supports selection, aggregation, filtering, group by, order by, distinct queries on data
+      </>
+    ),
+  },
+  {
+    title: 'Hybrid tables',
+    icon: 'list',
+    description: (
+      <>
+        Consist of of <Link to="/img/dynamic-table.png">both offline and realtime table</Link>. Use realtime table only to cover segments for which offline data may not be available yet
```

Comment:
does this link work? `/img/dynamic-table.png`

Reply:
Yes